docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+
+
+Other pipelines
+---------------
+In addtion to the Level 1, 2, and 3 pipelines discussed so far, there are 2 other 
+pipelines- Dark Processing, and Guide Star Processing.
+
+Dark Processing pipeline - calwebb_dark
++++++++++++++++++++++++++++++++++++++++
+The Dark Pipeline applies basic detector-level corrections to all dark exposures.
+For details on thie pipeline see `Calwebb_dark
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_dark.html>`_
+All of the steps in this pipeline are group_scale, dq_init, saturation, ipc, superbias, 
+refpix, linearity, and rcsd.  The only one of these steps that either uses or
+modifies variance/error arrays that are propagated through the pipeline is
+group_scale, described here:
+
+Group_scale
+~~~~~~~~~~~
+This step is applied after the ramp_fitting step, and applies to both the rate and
+rateints products from ramp_fit. The correction is applied to the ERR, and will
+also be applied to the VAR_POISSON, and VAR_RNOISE arrays.  The details of the 
+calculations can be found at `Group Scale
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/group_scale/description.html>`_
+
+
+Guide Star Processing pipeline - calwebb_guider
++++++++++++++++++++++++++++++++++++++++++++++++
+The Guider pipeline is only for use with data resulting from the FGS guiding functions.
+The steps in this pipeline are dq_init, guider_cds, and flat_field.
+ 
+The only one of these steps that either uses or modifies variance/error arrays
+that are propagated through the pipeline is flat_field, described here:
+
+Flat_field
+~~~~~~~~~~
+{NOTE get from the latest online readthedocs for the step once James updates it.}
+The SCI array from the flat-field reference file is divided into the VAR_POISSON
+and VAR_RNOISE arrays of the science data set. The variance in the flat is
+calculated in this step; the ERR array is then recalculated as these three
+variances added in quadrature.  The newly created VAR_FLAT array and the updated
+VAR_POISSON and VAR_RNOISE are output to the science products(s).  For details of
+the flat fielding correction, see `Flatfielding
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/flatfield/index.html>`_
+
+
+The table below is a summary of which steps create and output arrays related to 
+the cumulative error, modify and output the arrays, or use locally but do not
+output modified arrays, and which level pipelines each step is in.
+
+
+================= ======================= ====================================== =========================== ==========================
+STEP              Creates arrays          Modifies arrays                        Step-specific use of arrays  Pipeline Level
+================= ======================= ====================================== =========================== ==========================
+photom            None                    ERR, VAR_POISSON, VAR_RNOISE, VAR_FLAT    None                     Stages 2,3
```

Comment:
I like the concept of this table. My only comment is that I think it would make more logical sense to order the rows from top to bottom based on the actual order of execution through the pipelines. So this means putting "ramp_fit" at the top, followed by "gain_scale", followed by all the image2/spec2 steps, and finally the stage 3 steps.